eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Binary/Bugs.hs

```diff
@@ -0,0 +1,57 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Cardano.Ledger.Conway.Binary.Bugs where
+
+import Cardano.Ledger.Binary (decCBOR, decodeFullAnnotatorFromHexText)
+import Cardano.Ledger.Conway.Core (EraTx (..), eraProtVerLow)
+import Test.Cardano.Ledger.Common (NFData, Spec, describe, expectRightDeep_, it)
+
+spec ::
+  forall era.
+  ( EraTx era
+  , NFData (Tx era)
+  ) =>
+  Spec
+spec = describe "Bugs" $ do
```

Comment:
I would rather call it a regression test (or even a golden), since there aren't any bugs here.
```suggestion
spec = describe "Regression" $ do
```